scripts/is/intg/run-int-test.sh

```
@@ -0,0 +1,289 @@
+zip -q -r $TESTGRID_DIR/$PRODUCT_NAME-$PRODUCT_VERSION.zip $PRODUCT_NAME-$PRODUCT_VERSION
+
+log_info "Navigating to integration test module directory"
+ls $INT_TEST_MODULE_DIR
+
+# Check for master branch execution or tag-based execution
+if [[ "$PRODUCT_VERSION" != *"SNAPSHOT"* ]]; then
+    cd $TESTGRID_DIR/$PRODUCT_REPOSITORY_NAME || log_error "Failed to navigate to product repository directory"
+    echo $JAVA_HOME
+    #If support add the nexus repository to the pom.xml
+    if [[ "$PRODUCT_REPOSITORY_BRANCH" == *"support"* ]]; then
+        cp $TESTGRID_DIR/add_patch_repository.sh $TESTGRID_DIR/$PRODUCT_REPOSITORY_NAME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)